tox.ini

```diff
@@ -7,14 +7,22 @@
 envlist = py27, py35, py36, py37, py38, {py35, py36, py37, py38}-async
 
 [testenv]
+extras = async: async
+
 commands = coverage erase
            !async: coverage run -m unittest discover -p "test_*.py" []
-           async:  coverage run -m unittest discover -p "*test_*.py" []
-           !py27:  coverage report
-           py27:   coverage report --ignore-errors
+           async: coverage run -m unittest discover -p "*test_*.py" []
+           coverage report --omit="sendgrid/async_sendgrid.py"i
 
 deps = -rrequirements.txt
        coverage
        async: asynctest
-       async: python-http-client[async]==3.2.6dev5
-       py27: mock
+
+
+[testenv:py27]
+commands = coverage erase
+           coverage run -m unittest discover -p "test_*.py" []
+           coverage report --omit="sendgrid/async_sendgrid.py"
+
+deps = {[testenv]deps}
+       mock
```

Comment:
If `commands` go out this may also be set as conditional dependency in main section.